Ensure message endpoint is annotated with EnabledBy

szewai opened the pull request:
#### c037533251266302ea3a55a6f0ef51102e9ed2cd
<pre>
Ensure message endpoint is annotated with EnabledBy
<a href="https://bugs.webkit.org/show_bug.cgi?id=284654">https://bugs.webkit.org/show_bug.cgi?id=284654</a>
<a href="https://rdar.apple.com/141460474">rdar://141460474</a>

Reviewed by Ryosuke Niwa.

Force a compile error if a message endpoint is added to non-WebContent process without EnabledBy (or
ExceptionForEnabledBy) annotation.

* Source/WebKit/GPUProcess/GPUConnectionToWebProcess.messages.in:
* Source/WebKit/GPUProcess/GPUProcess.messages.in:
* Source/WebKit/GPUProcess/RemoteSharedResourceCache.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteBarcodeDetector.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteFaceDetector.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteTextDetector.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteImageBuffer.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteImageBufferSet.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.messages.in:
* Source/WebKit/GPUProcess/graphics/wc/RemoteWCLayerTreeHost.messages.in:
* Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaEngineConfigurationFactoryProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaPlayerProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaResourceManager.messages.in:
* Source/WebKit/GPUProcess/media/RemoteRemoteCommandListenerProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.messages.in:
* Source/WebKit/GPUProcess/webrtc/RemoteAudioMediaStreamTrackRendererInternalUnitManager.messages.in:
* Source/WebKit/ModelProcess/ModelProcess.messages.in:
* Source/WebKit/NetworkProcess/Authentication/AuthenticationManager.messages.in:
* Source/WebKit/NetworkProcess/Cookies/WebCookieManager.messages.in:
* Source/WebKit/NetworkProcess/CustomProtocols/LegacyCustomProtocolManager.messages.in:
* Source/WebKit/NetworkProcess/NetworkBroadcastChannelRegistry.messages.in:
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkContentRuleListManager.messages.in:
* Source/WebKit/NetworkProcess/NetworkProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkResourceLoader.messages.in:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.messages.in:
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerConnection.messages.in:
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.messages.in:
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkMDNSRegister.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCMonitor.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.messages.in:
* Source/WebKit/NetworkProcess/webrtc/RTCDataChannelRemoteManagerProxy.messages.in:
* Source/WebKit/Scripts/generate-derived-log-sources.py:
(generate_messages_file):
* Source/WebKit/Scripts/webkit/messages.py:
(async_message_statement):
* Source/WebKit/Scripts/webkit/model.py:
(MessageReceiver.__init__):
(Message.__init__):
(generate_global_model):
* Source/WebKit/Scripts/webkit/parser.py:
(parse):
* Source/WebKit/Scripts/webkit/tests/TestWithCVPixelBuffer.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithDispatchedFromAndTo.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithEnabledBy.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithIfMessage.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithImageData.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithLegacyReceiver.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSemaphore.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStream.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamBatched.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamBuffer.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamServerConnectionHandle.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclass.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclassAndWantsAsyncDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclassAndWantsDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsAsyncDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsDispatchNoSyncMessages.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithoutAttributes.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithoutUsingIPCConnection.messages.in:
* Source/WebKit/Shared/API/Cocoa/RemoteObjectRegistry.messages.in:
* Source/WebKit/Shared/AuxiliaryProcess.messages.in:
* Source/WebKit/Shared/IPCConnectionTester.messages.in:
* Source/WebKit/Shared/IPCStreamTester.messages.in:
* Source/WebKit/Shared/IPCStreamTesterProxy.messages.in:
* Source/WebKit/Shared/IPCTester.messages.in:
* Source/WebKit/Shared/IPCTesterReceiver.messages.in:
* Source/WebKit/Shared/LogStream.messages.in:
* Source/WebKit/UIProcess/Cocoa/PlaybackSessionManagerProxy.messages.in:
* Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.messages.in:
* Source/WebKit/UIProcess/Downloads/DownloadProxy.messages.in:
* Source/WebKit/UIProcess/DrawingAreaProxy.messages.in:
* Source/WebKit/UIProcess/Extensions/WebExtensionContext.messages.in:
* Source/WebKit/UIProcess/Extensions/WebExtensionController.messages.in:
* Source/WebKit/UIProcess/GPU/GPUProcessProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/RemoteWebInspectorUIProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/WebInspectorUIExtensionControllerProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.messages.in:
* Source/WebKit/UIProcess/Model/ModelProcessProxy.messages.in:
* Source/WebKit/UIProcess/Network/CustomProtocols/LegacyCustomProtocolManagerProxy.messages.in:
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.messages.in:
* Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.messages.in:
* Source/WebKit/UIProcess/UserContent/WebUserContentControllerProxy.messages.in:
* Source/WebKit/UIProcess/ViewGestureController.messages.in:
* Source/WebKit/UIProcess/VisitedLinkStore.messages.in:
* Source/WebKit/UIProcess/WebGeolocationManagerProxy.messages.in:
* Source/WebKit/UIProcess/WebPageProxy.messages.in:
* Source/WebKit/UIProcess/WebPasteboardProxy.messages.in:
* Source/WebKit/UIProcess/WebPermissionControllerProxy.messages.in:
* Source/WebKit/UIProcess/WebProcessPool.messages.in:
* Source/WebKit/UIProcess/WebProcessProxy.messages.in:
* Source/WebKit/UIProcess/dmabuf/AcceleratedBackingStoreDMABuf.messages.in:
* Source/WebKit/UIProcess/ios/SmartMagnificationController.messages.in:
* Source/WebKit/UIProcess/ios/WebDeviceOrientationUpdateProviderProxy.messages.in:
* Source/WebKit/UIProcess/mac/SecItemShimProxy.messages.in:
* Source/WebKit/WebProcess/Geolocation/WebGeolocationManager.messages.in:
* Source/WebKit/WebProcess/Model/ModelProcessConnection.messages.in:
* Source/WebKit/WebProcess/Model/ModelProcessModelPlayer.messages.in:
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.messages.in:
* Source/WebKit/WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf.messages.in:
* Source/WebKit/WebProcess/glib/SystemSettingsManager.messages.in:
* Source/WebKit/WebProcess/glib/UserMediaCaptureManager.messages.in:
* Source/WebKit/webpushd/PushClientConnection.messages.in:

Canonical link: <a href="https://commits.webkit.org/287923@main">https://commits.webkit.org/287923@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f2dc868ca6102355659e9c4698972ba52a82e19a

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/81308 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/833 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/35251 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/85837 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/32294 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/851 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/8649 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/63475 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/21240 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/84377 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/581 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/73999 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/43770 "Passed tests") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/80765 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/479 "Passed tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/30752 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/71973 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/28742 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/87272 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/8538 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/6069 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/71782 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/8719 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/69829 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/71015 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/17686 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/15072 "Passed tests") | [⏳ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-WK2-Intel-Tests-EWS "Waiting to run tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/8500 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/14023 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/8336 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/11857 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/10144 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rniwa commented on Source/WebKit/Scripts/webkit/parser.py:
```diff
@@ -123,13 +130,19 @@ def parse(file):
                     coalescing_key_indices = parse_coalescing_keys(match.group(1), [parameter.name for parameter in parameters])
 
+            if enabled_by and enabled_by_exception:
+                raise Exception("ERROR: 'ExceptionForEnabledBy' cannot be use together with 'EnabledBy=%s'" % enabled_by)
```
Nit: cannot be use**d**.

rniwa reviewed (approved):
We should probably add a style checker change to warn people of new use of ExceptionForEnabledBy.

szewai committed:
Ensure message endpoint is annotated with EnabledBy
https://bugs.webkit.org/show_bug.cgi?id=284654
rdar://141460474

Reviewed by Ryosuke Niwa.

Force a compile error if a message endpoint is added to non-WebContent process without EnabledBy (or
ExceptionForEnabledBy) annotation.

* Source/WebKit/GPUProcess/GPUConnectionToWebProcess.messages.in:
* Source/WebKit/GPUProcess/GPUProcess.messages.in:
* Source/WebKit/GPUProcess/RemoteSharedResourceCache.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteBarcodeDetector.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteFaceDetector.messages.in:
* Source/WebKit/GPUProcess/ShapeDetection/RemoteTextDetector.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteImageBuffer.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteImageBufferSet.messages.in:
* Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.messages.in:
* Source/WebKit/GPUProcess/graphics/wc/RemoteWCLayerTreeHost.messages.in:
* Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaEngineConfigurationFactoryProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaPlayerProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteMediaResourceManager.messages.in:
* Source/WebKit/GPUProcess/media/RemoteRemoteCommandListenerProxy.messages.in:
* Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.messages.in:
* Source/WebKit/GPUProcess/webrtc/RemoteAudioMediaStreamTrackRendererInternalUnitManager.messages.in:
* Source/WebKit/ModelProcess/ModelProcess.messages.in:
* Source/WebKit/NetworkProcess/Authentication/AuthenticationManager.messages.in:
* Source/WebKit/NetworkProcess/Cookies/WebCookieManager.messages.in:
* Source/WebKit/NetworkProcess/CustomProtocols/LegacyCustomProtocolManager.messages.in:
* Source/WebKit/NetworkProcess/NetworkBroadcastChannelRegistry.messages.in:
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkContentRuleListManager.messages.in:
* Source/WebKit/NetworkProcess/NetworkProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkResourceLoader.messages.in:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.messages.in:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.messages.in:
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerConnection.messages.in:
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.messages.in:
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkMDNSRegister.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCMonitor.messages.in:
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.messages.in:
* Source/WebKit/NetworkProcess/webrtc/RTCDataChannelRemoteManagerProxy.messages.in:
* Source/WebKit/Scripts/generate-derived-log-sources.py:
(generate_messages_file):
* Source/WebKit/Scripts/webkit/messages.py:
(async_message_statement):
* Source/WebKit/Scripts/webkit/model.py:
(MessageReceiver.__init__):
(Message.__init__):
(generate_global_model):
* Source/WebKit/Scripts/webkit/parser.py:
(parse):
* Source/WebKit/Scripts/webkit/tests/TestWithCVPixelBuffer.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithDispatchedFromAndTo.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithEnabledBy.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithIfMessage.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithImageData.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithLegacyReceiver.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSemaphore.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStream.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamBatched.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamBuffer.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithStreamServerConnectionHandle.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclass.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclassAndWantsAsyncDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithSuperclassAndWantsDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsAsyncDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsDispatch.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithWantsDispatchNoSyncMessages.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithoutAttributes.messages.in:
* Source/WebKit/Scripts/webkit/tests/TestWithoutUsingIPCConnection.messages.in:
* Source/WebKit/Shared/API/Cocoa/RemoteObjectRegistry.messages.in:
* Source/WebKit/Shared/AuxiliaryProcess.messages.in:
* Source/WebKit/Shared/IPCConnectionTester.messages.in:
* Source/WebKit/Shared/IPCStreamTester.messages.in:
* Source/WebKit/Shared/IPCStreamTesterProxy.messages.in:
* Source/WebKit/Shared/IPCTester.messages.in:
* Source/WebKit/Shared/IPCTesterReceiver.messages.in:
* Source/WebKit/Shared/LogStream.messages.in:
* Source/WebKit/UIProcess/Cocoa/PlaybackSessionManagerProxy.messages.in:
* Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.messages.in:
* Source/WebKit/UIProcess/Downloads/DownloadProxy.messages.in:
* Source/WebKit/UIProcess/DrawingAreaProxy.messages.in:
* Source/WebKit/UIProcess/Extensions/WebExtensionContext.messages.in:
* Source/WebKit/UIProcess/Extensions/WebExtensionController.messages.in:
* Source/WebKit/UIProcess/GPU/GPUProcessProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/RemoteWebInspectorUIProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/WebInspectorUIExtensionControllerProxy.messages.in:
* Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.messages.in:
* Source/WebKit/UIProcess/Model/ModelProcessProxy.messages.in:
* Source/WebKit/UIProcess/Network/CustomProtocols/LegacyCustomProtocolManagerProxy.messages.in:
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.messages.in:
* Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.messages.in:
* Source/WebKit/UIProcess/UserContent/WebUserContentControllerProxy.messages.in:
* Source/WebKit/UIProcess/ViewGestureController.messages.in:
* Source/WebKit/UIProcess/VisitedLinkStore.messages.in:
* Source/WebKit/UIProcess/WebGeolocationManagerProxy.messages.in:
* Source/WebKit/UIProcess/WebPageProxy.messages.in:
* Source/WebKit/UIProcess/WebPasteboardProxy.messages.in:
* Source/WebKit/UIProcess/WebPermissionControllerProxy.messages.in:
* Source/WebKit/UIProcess/WebProcessPool.messages.in:
* Source/WebKit/UIProcess/WebProcessProxy.messages.in:
* Source/WebKit/UIProcess/dmabuf/AcceleratedBackingStoreDMABuf.messages.in:
* Source/WebKit/UIProcess/ios/SmartMagnificationController.messages.in:
* Source/WebKit/UIProcess/ios/WebDeviceOrientationUpdateProviderProxy.messages.in:
* Source/WebKit/UIProcess/mac/SecItemShimProxy.messages.in:
* Source/WebKit/WebProcess/Geolocation/WebGeolocationManager.messages.in:
* Source/WebKit/WebProcess/Model/ModelProcessConnection.messages.in:
* Source/WebKit/WebProcess/Model/ModelProcessModelPlayer.messages.in:
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.messages.in:
* Source/WebKit/WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf.messages.in:
* Source/WebKit/WebProcess/glib/SystemSettingsManager.messages.in:
* Source/WebKit/WebProcess/glib/UserMediaCaptureManager.messages.in:
* Source/WebKit/webpushd/PushClientConnection.messages.in:

Canonical link: https://commits.webkit.org/287923@main
